hs-bindgen/src/HsBindgen/Hs/AST/Type.hs

```diff
@@ -21,6 +21,7 @@ data HsPrimType
     | HsPrimCUShort
     | HsPrimCLong
     | HsPrimCULong
+    | HsPrimCString
```

Comment:
CString is `Ptr Char`. I'm not sure we need to introduce another way to encode it. (It's not even a newtype).